provider/src/main/java/com/raygun/raygun4android/RaygunSettings.java

```diff
@@ -62,7 +62,7 @@ public static HashSet<String> getIgnoredURLs() {
         return ignoredURLs;
     }
 
-    static HashSet<String> getIgnoredViews() {
+    public static HashSet<String> getIgnoredViews() {
```

Comment:
Since the `RUM` class is moved to the `.rum` package, it cannot access `getIgnoredViews` per package visibility anymore.